app/webhook.go

```diff
@@ -353,11 +353,12 @@ func (a *App) UpdateIncomingWebhook(oldHook, updatedHook *model.IncomingWebhook)
 	updatedHook.TeamId = oldHook.TeamId
 	updatedHook.DeleteAt = oldHook.DeleteAt
 
-	if result := <-a.Srv.Store.Webhook().UpdateIncoming(updatedHook); result.Err != nil {
-		return nil, result.Err
+	result, err := a.Srv.Store.Webhook().UpdateIncoming(updatedHook)
```

Comment:
This is no longuer a "result" now is a webhook I think renamed it to other thing like, newWebhook, or savedWebhook would be clearer.